README.md

```diff
@@ -9,31 +9,29 @@ yarn add react-waterfall
 Here is an example of a basic implementation.
 
 ```js
-import { initStore } from 'react-waterfall'
+import { initStore, connect, Provider } from 'react-waterfall';
```

Comment:
It seems your prettier is not configured to use the project config 

https://github.com/didierfranc/react-waterfall/blob/master/.prettierrc